src/screens/auth/phone-auth/phone-auth-form.tsx

```diff
@@ -32,96 +36,130 @@ const renderCountrySelectMenu = (
-              onClose();
-            }}
-          >
-            {option.label}
-          </Menu.Item>
-        ))}
-      </ScrollView>
-    </Menu>
-  );
-};
+          {option.label}
+        </Menu.Item>
+      ))}
+    </ScrollView>
+  </Menu>
+);
 
 const PhoneAuthForm: React.FC<PhoneAuthFormProps> = ({ onSuccess, onError }) => {
+  const themeColor = useThemeColor('primary.500');
+
+  const styles = usePhoneAuthFormStyles();
   const { formik, isSendOTPLoading } = usePhoneAuthForm({
     onSendOTPSuccess: onSuccess,
     onError: onError,
   });
 
   const { isOpen, onOpen, onClose } = useDisclose();
+  const [isChecked, setIsChecked] = useState(false);
 
   const handleSelectChange = (value: string) => {
     const [countryCode, country] = value.split(', ');
     formik.setFieldValue('countryCode', countryCode);
     formik.setFieldValue('country', country);
   };
 
+  const phoneNumber = new PhoneNumber({
+    countryCode: formik.values.countryCode.replace('+', ''),
+    phoneNumber: formik.values.phoneNumber,
+  });
+
+  const phoneNumberPlaceholder = new PhoneNumber({
+    countryCode: formik.values.countryCode.replace('+', ''),
+    phoneNumber: '9999999999',
+  }).getFormattedWithoutCountryCode();
+
   return (
-    <>
-      <VStack space={6} flex={1}>
+    <Box flex={1} pb={4}>
+      <VStack space={6} flex={1} mb={8}>
         <Container>
-          <Heading size="lg">Welcome</Heading>
-          <Heading mt="1" size="xs">
-            Enter your number to continue
-          </Heading>
+          <Heading size="lg">Login</Heading>
         </Container>
-        <FormControl
-          isRequired={true}
-          isInvalid={formik.touched.phoneNumber && Boolean(formik.errors.phoneNumber)}
-        >
-          <FormControl.Label>Phone Number</FormControl.Label>
-          <HStack space={2}>
+        <FormControl label={'Mobile Number'}>
+          <HStack space={2} width="100%">
             {renderCountrySelectMenu(formik, isOpen, onOpen, onClose, handleSelectChange)}
-            <Input
-              value={formik.values.phoneNumber}
-              onChangeText={formik.handleChange('phoneNumber')}
-              keyboardType="numeric"
+            <Box
               flex={3}
-              placeholder="XXXXXXXXXX"
-            />
+              justifyContent="center"
+              style={[
+                styles.inputBox,
+                formik.touched.phoneNumber && formik.errors.phoneNumber ? styles.errorStyle : {},
+              ]}
+            >
+              <Input
+                value={phoneNumber.getFormattedWithoutCountryCode()}
+                onChangeText={formik.handleChange('phoneNumber')}
+                keyboardType="numeric"
+                placeholder={phoneNumberPlaceholder}
+              />
+            </Box>
           </HStack>
-          <FormControl.ErrorMessage>{formik.errors.phoneNumber}</FormControl.ErrorMessage>
+          {formik.touched.phoneNumber && formik.errors.phoneNumber ? (
```

Comment:
In an ideal world, the text message should be displayed by `FormControl` so not every component has to do this.

Reply:
Noted.